fetch-builds-by-date-range-from-all-connected-controllers.groovy

```diff
@@ -0,0 +1,26 @@
+import com.cloudbees.opscenter.server.clusterops.steps.MasterGroovyClusterOpStep
+
+def retour = ''
+def stream = new ByteArrayOutputStream();
+def listener = new StreamBuildListener(stream);
+
+Jenkins.instance.getAllItems(com.cloudbees.opscenter.server.model.ConnectedMaster).each{
+    it.channel.call(new MasterGroovyClusterOpStep.Script("""
+          import jenkins.model.Jenkins
+          import java.time.Instant
+
+          // dates should use the ISO8601 format
+          def startDate = Instant.parse("2022-06-24T16:10:43.000Z")
+          def endDate = Instant.now()
+
+          def builds = Jenkins.instance.getAllItems()
+                  .findAll { it instanceof Job }
+                  .collect { it.builds.byTimestamp(startDate.toEpochMilli(), endDate.toEpochMilli()) }
+                  .flatten()
+
+          return builds
+    """, listener, "host-script.groovy", [:]))
+    retour = stream.toString().minus('Result: ')
```

Comment:
the variable is overridden after loop increment. In other words, it looks like only the output from the last controller in the loop will be displayed.. 

```suggestion
    retour = retour << stream.toString().minus('Result: ').minus('\n')
```